### PR TITLE
Make Dataflow install instructions more inclusive

### PR DESCRIPTION
**Pull request by svick**
TPL Dataflow does not require .Net Framework or VS, so I have made the instructions more inclusive by not mentioning .Net Framework explicitly and by adding a command to install it using the .Net Core CLI.

### CODE REVIEW

File: includes/tpl-install-instructions.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 > [!NOTE]
-> The TPL Dataflow Library (the <xref:System.Threading.Tasks.Dataflow> namespace) is not distributed with the .NET Framework. To install the <xref:System.Threading.Tasks.Dataflow> namespace, open your project in Visual Studio, choose **Manage NuGet Packages** from the **Project** menu, and search online for the `System.Threading.Tasks.Dataflow` package.
+> The TPL Dataflow Library (the <xref:System.Threading.Tasks.Dataflow> namespace) is not distributed with .NET. To install the <xref:System.Threading.Tasks.Dataflow> namespace in Visual Studio, choose **Manage NuGet Packages** from the **Project** menu, and search online for the `System.Threading.Tasks.Dataflow` package. Alternatively, to install it using [the .Net Core CLI](~/docs/core/tools/index.md), run `dotnet add package System.Threading.Tasks.Dataflow`.
```
Comment (rpetrusha) [inline includes/tpl-install-instructions.md]: The **Manage NuGet Packages** option is only available if a project is open. Removing the phrase creates the impression that the TPL Dataflow Library can be installed without a project being open.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @svick, for revising this text so that it applies to .NET Core as well as .NET Framework. It would be good to add back the phrase about opening a project, though.

File: includes/tpl-install-instructions.md (hunk 1 above)
Comment (svick) [reply includes/tpl-install-instructions.md]: @rpetrusha Done.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you so much for making the additional change, @svick. We'll merge your PR as soon as the build completes successfully.

**Pull request merged**